src/macros/middleware.rs

```diff
@@ -43,7 +44,10 @@
 /// supports annotating the macro so as to drive the inference allowing the handler
 /// code to remain with minimal annotations.
 ///
-/// ```
+/// # ignoring this, since the middleware macro needs work
+/// # TODO: migration cleanup - fix middleware macro, or remove it if closures
+/// #       can be made to take its place
+/// ```rust,ignore
```

Comment:
Do doc comments ignore `/// #`?  My cursory research doesn't indicate that they do; perhaps the "ignore" comments should be `//` instead?